stylelint-polaris/index.js

```diff
@@ -186,6 +200,7 @@ module.exports = {
           /--p-range-slider-thumb-size-base/,
           /--p-range-slider-thumb-size-active/,
           /--p-override-visible/,
+          /--p-override-loading-z-index/,
```

Comment:
Moved this legacy custom property from depth to layout since all other `z-index` rules are categorized there.